### DISCUSSION TITLE
Please tell me about the rss is not correct in download website incould the magnet link url, thanks~~

### DISCUSSION THREAD

**Discussion by DFP86** [FreshRSS/FreshRSS#5931] [Q&A] [answered]
hello, I am a newcomer. After I install freshRSS(liunxserver/freshRSS) docker in my nas I find I can't add a new RSS feed in a media download site. (Maybe the RSS is only use for download torrents.)

The RSS is a xml. Including the "enclosure url=magnet?:xt=XXXXXX". When I add this xml link FreshRSS can't import it. I click chcek button, the result said: url must be a full URL: magnet:?xxxxx.  And said: Invalid email address: GMTeam:<author><![CDATA[GMTeam]]></author>. The import Can not continue.
![image](https://github.com/FreshRSS/FreshRSS/assets/12046701/0e7cd33c-4380-421f-a099-28fea8e9362b)

I don't want to download the torrent file. I  just want to see the AnimeTV show update the last new one or not. Because I have a lot of serial to watch. I just want to see the information update.

So I search the  issues and discussion, and read the User documentation. But I can not search everthing about this...
Please tell me is FreshRSS not support this type Rss? Or just me not set the right setting? Thanks a lot!

**Comment by DFP86** [comment, marked as answer]
Oh... I just ... Is work!!!!!!

Don't select xml+xpatch, just select atom.  So nice~

**Reply by Frenzie** [reply]
Ah, that's sneaky. ;-) I was going to say the validity shouldn't matter too much (in the context shown here) as long as it's well-formed, leaving me at a loss.
